datasets/reuters21578/reuters21578.py

```diff
@@ -0,0 +1,326 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors and the HuggingFace NLP Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+"""Reuters 21578"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import os
+
+import nlp
+
+
+_CITATION = """\
```

Comment:
Could you format the citation as usual ?

Reply:
Thanks for the formatting 👍 

There are still a few things we could do. As the citation is rendered in the viewer and on the website we should pay attention that it's all beautiful x)
- make sure that the citations have the same indentation
- put the commas at the end of the lines
- place the = character in a consistent position (sometimes it's vertically aligned, sometimes not)
- remove empty spaces before the `}` (see the 6th citation for example)